config/local-setup/workloads/mt-ingress.yaml

```
@@ -64,6 +64,24 @@ spec:
     app.kubernetes.io/name: marin3r
   type: LoadBalancer
 
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: mt-ingress
+spec:
+  podMetricsEndpoints:
+    - interval: 30s
```

```suggestion
    - interval: 60s
```

That's a lot of metrics and this is supposed to be able to run locally

---

Sure, I just copy/paste the resource from SaaS

---

fixed! https://github.com/3scale-sre/saas-operator/pull/323/commits/b8768d81e24e9edd1a95f2fdb26679c1a111cb2c